From: Reinhard Tartler <siretart@tauware.de>
Date: Wed, 25 May 2022 18:22:00 +0800
Subject: Fixup live.cfg for casper

Last-Update: 2012-08-20
live.cfg from the themes package include a 'boot=live' parameter, which works only
for live-config. casper OTOH requires 'boot=casper'
---
 scripts/build/lb_binary_syslinux | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/scripts/build/lb_binary_syslinux b/scripts/build/lb_binary_syslinux
index 300841a..3b7bedb 100755
--- a/scripts/build/lb_binary_syslinux
+++ b/scripts/build/lb_binary_syslinux
@@ -254,6 +254,13 @@ then
 	sed -i  -e "s|@LB_BOOTAPPEND_LIVE@|${LB_BOOTAPPEND_LIVE}|g" \
 		-e "s|@LB_BOOTAPPEND_FAILSAFE@|${LB_BOOTAPPEND_FAILSAFE}|g" \
 	${_TARGET}/live.cfg
+
+	case "${LB_INITRAMFS}" in
+		casper)
+			sed -i -e "s|boot=live|boot=casper|" \
+			    ${_SUFFIX}/live.cfg
+			;;
+	esac
 fi
 
 if [ -e ${_TARGET}/install.cfg ]
